api/pkg/db/migration/202008071700_add_migration_on_catalog.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"github.com/jinzhu/gorm"
+	"go.uber.org/zap"
+	"gopkg.in/gormigrate.v1"
+
+	"github.com/tektoncd/hub/api/pkg/db/model"
+)
+
+func addMigrationOnCatalog(log *zap.SugaredLogger) *gormigrate.Migration {
```

Comment:
I think Migration don't need to be in the name, could be what the migration does?